test/integration/enumerate_databases.test.ts

```diff
@@ -0,0 +1,121 @@
+import { expect } from 'chai';
+
+import { AddUserOptions, MongoClient, MongoServerError } from '../../src';
+
+describe('listDatabases', function () {
+  let client: MongoClient;
+
+  beforeEach(async function () {
+    client = this.configuration.newClient();
+    await client.connect();
+  });
+
+  afterEach(async function () {
+    await client?.close();
+  });
+
+  it('should return an Array', async () => {
+    const dbInfo = await client.db().admin().listDatabases();
+    expect(Array.isArray(dbInfo.databases)).to.be.true;
```

Comment:
does `to.be.an('array')` not work here?

Reply:
TIL! I thought this only took `typeof` results.